FirebaseVertexAI/Sources/Types/Public/Schema.swift

```diff
@@ -20,19 +20,38 @@ import Foundation
 /// [OpenAPI 3.0 schema object](https://spec.openapis.org/oas/v3.0.3#schema).
 public class Schema {
   /// Modifiers describing the expected format of a string `Schema`.
-  public enum StringFormat {
+  public struct StringFormat: EncodableProtoEnum {
+    enum Kind: String {
+      case custom // TODO: Remove when we have at least one specific string format.
```

Comment:
Won't it be a breaking change to remove?

Reply:
Fortunately not, the `Kind` enum is internal.

Reply:
Hmm, I see `custom` being used in the integration test?

Reply:
I believe the `custom` is coming from:
```swift
    public static func custom(_ format: String) -> StringFormat {
      return self.init(rawValue: format)
    }
```
Because the remaining `Kind.custom` case does not take an associated value, but the public static function `custom` does.

Reply:
Exactly that, @ncooke3! I'm using that pattern to mimic the old `case custom(String)` since it has an associated value. To avoid confusion with the static method, what if I rename this case to `unused`? I painted myself into a corner with: https://github.com/firebase/firebase-ios-sdk/blob/ae484d301630831808f8a9c31922d2d502e30d55/FirebaseVertexAI/Sources/Protocols/Internal/CodableProtoEnum.swift#L30
Rather than refactor `ProtoEnum` for this edge case, I think it's best to just provide some placeholder enum and Swift requires it to have at least one case.

Reply:
The other alternatives I can think of are to resurrect the `extension Schema.StringFormat: RawRepresentable {` and make this `case custom(String)` again or to not make it an `EncodableProtoEnum` and write a custom `Encodable` implementation here.

Reply:
> Rather than refactor `ProtoEnum` for this edge case, I think it's best to just provide some placeholder enum and Swift requires it to have at least one case.

An `unused` case sounds reasonable to me.

Reply:
I definitely prefer the `unused` case approach. Updated the PR and added some comments so future me can understand what was going on. Thanks @paulb777 for calling out that it was confusing and @ncooke3 for working through it!